web/src/test/java/se/inera/intyg/webcert/web/service/facade/impl/UserServiceImplTest.java

```diff
@@ -81,15 +88,15 @@ void setUp() {
             .when(user)
             .getNamn();
 
-        doReturn(getNavigableCareProvider())
+        lenient().doReturn(getNavigableCareProvider())
```

Comment:
I think it is better to avoid using lenient and instead restructure the tests in different @Nested classes with separate mocking.